language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/AbstractCompletionProvider.java

```diff
@@ -302,6 +302,10 @@ private List<LSCompletionItem> getTypeItems(BallerinaCompletionContext context)
                 String label = completionItem.getLabel();
                 String moduleName = importDeclarationNode.prefix().isEmpty() ? moduleSymbol.id().moduleName()
                         : importDeclarationNode.prefix().get().prefix().text();
+                if (moduleName.contains(".")) {
+                    String[] modNameParts = moduleName.split("\\.");
+                    moduleName = modNameParts[modNameParts.length - 1];
```

Comment:
Shall we add a test case covering this

Reply:
@mindula can you please check this and comment/fix.

Reply:
@malinthar I have already added a test case for this.

Reply:
Updated the test case